docs/Course Wiki/PHYS Course Offerings/PHYS211.md

```diff
@@ -0,0 +1,50 @@
+# PHYS211
+
+PHYS211 (University Physics: Mechanics) is a 4-credit hour course that is required for both EEs and CEs as a part of the Foundational Mathematics and Science requirement. It is offered in the fall, spring and summer semesters.
+
+## Content Covered
+
+- Kinematics
+- Relative and Circurlar motion
+- Newton's Laws
+- Forces, Friction
+- Work & Energy
+- Center of Mass
+- Momentum
+- Collisions, Impulse and Reference Frames
+- Ratations, Torque, Angular Momentum
+- Simple Harmonic Motion, Pendula, Waves
+- Fluid Statics, Fluid Dynamics
+
+## Prerequisites
+
+- [MATH231](MATH231.md) (corequisite)
+
+The official prerequisites to PHYS211 is credit or concurrent registration in [MATH231](MATH231.md)(While MATH231 is listed as a corequisite for PHYS211, no heavy calculation will be used in PHYS211).
+
+## When to Take It
+
+ECE majors typically take PHYS211 within the first two semesters, most commonly during the first semester.
+
+## Course Structure
+
+PHYS211 holds two lectures each week. There are prelectures and checkpoints before each lecture. Attendance for the lectures is required using iClickers. PHYS211 has a required discussion section, which meets weekly. During the discussion, students work on a problem set related to the topics covered in the lectures. At the end of the discussion, there is a weekly quiz except during the midterm weeks. There is also a required weekly lab section. During the lab, students work on a lab related to the topics covered in the lectures. There are prelabs due before the labs and a lab report due at the end of the lab session.
+
+## Instructors
+
+Recently, this course has been taught by Professor Tim Stelzer and Professor Jorge Leite Noronha Jr. Professor Tim Stelzer's lectures are rated very highly. However, do note that electronics are not allowed in lectures to take notes. 
+
+## Course Tips
+
+Between prelectures, lectures, labs and discussions, each topic will be covered multiple times. If students are confused about a certain topic, it is best to go through all the different components of the class. The formula sheet for this class is quite good. Understanding when to use each formula is very helpful for exams. The practice exams are very similar to the actual exams, they are a great tool to study. 
+
+## Life After
+
+Students will generally take [PHYS212](PHYS212.md) the semester after taking PHYS211. PHYS212 is not directly related to 211 but is required for both EEs and CEs. 
+
+## Infamous Topics
+
+Ratations and Waves can be difficult for students who are seeing them for the first time. Understanding of how to use the formulas on the formula sheet is extremely helpful for dealing with those topics in exams. 
```

Comment:
"Ratations and Waves" -> "Rotations and waves", "Understanding of" -> "Understanding"